EventListener/ControllerListener.php

```diff
@@ -50,13 +50,18 @@ public function __construct(Reader $reader)
      */
     public function onKernelController(FilterControllerEvent $event)
     {
-        if (!is_array($controller = $event->getController())) {
+        if (!is_array($controller = $event->getController()) && !is_object($controller)) {
```

Comment:
this won't deal properly with closures, which are objects (even though they are unlikely to be used in a fullstack project, and the bundle is unlikely to be used elsewhere)